---
layout: default
title: Sergei Turanov
---
<div class="blurb">
	<h1>Research</h1>
	<p>I’m an ichthyologist by training (graduated from the <a href="http://dalrybvtuz.ru/">Far Eastern
Technical Fisheries University</a>) 
	but my Candidate of Sciences degree (reffered to as <a href="https://en.wikipedia.org/wiki/Candidate_of_Sciences">equivalent
of PhD</a>) was stained in genetics so I almost lost the spirit of the classic ichthyology and moved toward the computational molecular 
	evolution approaches. That's a pity.<p>
	<footer>
    	</footer>
    	<p>There are several research directions I am happy to deal with.<p>
    	<h2>DNA barcoding</h2>
	<p><a href="http://www.barcodeoflife.org/content/about/what-dna-barcoding">DNA <span
class=SpellE>barcoding</span></a> is a global research program, which now represents one of the largest 
    	communities in the field of biological diversity investigation. The standardized molecular 
    	genetic techniques coupled with classic morphological methods with all of them relying on the 
    	global resource on the documentation of biological diversity (Barcode of Life Data System) 
    	made it possible to concentrate the efforts of biodiversity scientists from all over the world. 
    	Although we’ve got an independent node (<a href="http://www.imb.dvo.ru/misc/barcoding/index.htm">RUS-BOL</a>), DNA barcoding techniques are not actually popular 
    	in Russia and doubtfully will be amongst the high national priority research programs in opposite for 
    	instance to Canada, Germany or Norway. However, it is clear that Russia with its huge territory has 
    	one of the biggest sources of undocumented biological diversity and, hence, provides a great potential 
    	to accomplish the main goal of this program. 
    	The results we have obtained to date in this direction uncover some issues of species level taxonomic diversity 
    	among the perch-like (Actinopterygii: Perciformes) fishes from the Far Eastern Seas of Russia. 
    	The <i>Co-1</i> gene barcode reference library was assembled for 45 species. 
    	The analysis showed the existence of cryptic diversity 
    	(deeply diverged mitochondrial phylogroups within morphologically identical specimens) 
    	inside Lumpenus and Ammodytes genera. Furthermore, we found several pairs of putatively “young” species 
    	(morphologically distinct but genetically unresolved). 
    	That definitely would not be possible without kind help of our colleagues from the <a href="http://eng.kiost.ac/kordi_eng/main/"> Korea
Institute of Ocean Science and Technology</a> (Ansan) where we have obtained the nucleotide sequences.<p>
    	<h2>The patterns of origin of the taxonomic and phylogenetic diversity among forage fish species</h2>
    	<p><a href="https://en.wikipedia.org/wiki/Forage_fish">Forage fish</a> are small pelagic fish (suddenly), which are preyed on by larger predators for food. 
    	They are typically feed on plankton, mainly by filter feeding, and have some specific adaptations, 
    	which could be recognized in behavior (they form schools) and in the lack of 
    	(primary or repeated) adult stage when the switching to fish feeding should normally take place. 
    	The only group of the forage fish <a href="http://link.springer.com/article/10.1134%2FS1063074014060212">I have been studying to date</a> is Sand Lances (genus <i>Ammodytes</i>, by the way). 
    	The sand lances are small schooling fishes which occupy the shallow marine waters largely affecting the biomass of the coastal marine and terrestrial vertebrate species in the north hemisphere. 
    	The strict environmental specificity of sand lances (the preference of clean sandy habitats, pronounced stenothermy) 
    	resulted in the restricted gene flow among their different local populations and contributed to the origin of cryptic taxonomic diversity with the lack of discrete morphological characters but maintaining the clear genetic differences. 
    	Now I just want to go deeper in the investigation of phylogeographic patterns of the species diversity.<p>
    	<h2>Patterns of the structural evolution and phylogenetic informativeness of the mitochondrial control region sequences</h2>
	<p>The control region (CR) is a major non-protein-coding domain of mitochondrial DNA, 
	which contains transcription promoters for both strands of mitochondrial genome as well 
	as binding sites for transcription-replication machinery whereby the D-loop is normally formed, 
	a vital structure for the existence of mitochondrial genome as a biological replicator. 
	The origin and diversity of structure-functional elements of CR have been intensively investigated 
	with massive recent involvement of new diverse taxa. Within this direction 
	I’ve got but <a href="/Turanov MAPEEG 2015.pdf">have not actually published</a> yet some preliminary results on the analysis 
	of eelpouts (the fishes from infraorder Zoarcales) CR structure and variation as well as 
	discovered some apparent trends in the CR macroevolutionary patterns.<p>
</div><!-- /.blurb -->
